openedx/core/djangoapps/signals/signals.py

```diff
@@ -0,0 +1,9 @@
+"""
+This module contains all signals.
+"""
+
+from django.dispatch import Signal
+
+
+# Signal that fires when a user is graded (in lms/courseware/grades.py)
+MIN_GRADE_REQUIREMENT_STATUS = Signal(providing_args=["username", "grade_summary", "course_key", "deadline"])
```

Comment:
Minor: This signal gets fired when grades are calculated -- other apps may want to install receivers for this signal that do things other than calculate the minimum grade status for credit.  Please rename this to `GRADES_UPDATED` to make this clear.